microprot/tests/test_processing.py

```diff
@@ -0,0 +1,50 @@
+from unittest import TestCase, main
+from skbio.util import get_data_path
+from skbio import Sequence
+
+from microprot.scripts.processing import extract_sequences
+
+
+class ProcessingTests(TestCase):
+    def setUp(self):
+        self.input_faa = get_data_path('test_processing/input.faa')
+
+    def test_extract_sequences(self):
+        obs = extract_sequences(self.input_faa)
+        exp = []
+        x = Sequence('PIVQNLQGQMVHQAISPRTLNAWVKVVEEKAFSPEVIPMFSALSEGATPQDLNTML'
```

Comment:
more a general question than something specific to this code line: Do we care about strangely formatted fasta content, like arbitrary newlines, comments (a line starting with ;), different line lengths, ...